pyproject.toml

```diff
@@ -23,10 +23,11 @@ dependencies = [
     "numpy==2.0.2",
     "setuptools",
     "sympy==1.13.1",
-    "torch==2.5.0",
+    "torch (>=2.6.0,<3.0.0)",
     "triton",
     "typing-extensions==4.12.2",
-    "hydra-core==1.3.2"
+    "hydra-core==1.3.2",
+    "wandb (>=0.19.7,<0.20.0)"
```

Comment:
I recommend moving `wandb` to a different subcategory in the requirements. It should not be required for package development or inference but should only be included as a dependency for training. Similar to dev dependencies, you can define a separate category for training dependencies. This allows `wandb` to be included only for training while keeping it out of the core package, development, and inference dependencies.

Reply:
I agree.